src/framework/theme/components/select/select.component.html

```diff
@@ -28,6 +28,6 @@
 
 <nb-card *nbPortal class="select" [ngClass]="[status, overlayPosition]" [style.width.px]="hostWidth">
   <nb-card-body>
-    <ng-content select="nb-option, nb-option-group"></ng-content>
+    <ng-content select="nb-option, nb-option-group, ng-container"></ng-content>
```

Comment:
Not sure if it's ok to select `ng-container` here. This way it would be possible to pass arbitrary content into select. What do you think @aefox? Probably there is something we could do to restrict `ng-container` content (e.g. more specific selector)?

Reply:
@yggg I totally see your point on `possible to pass arbitrary content into select`. That indeed is a problem I did not figure out, nice catch 👍 

At the same time, please note the following `hack/trick`:
```
<nb-select placeholder="Select Showcase" [(selected)]="selectedItem">
  <ng-container *ngFor="let item of items" ngProjectAs="nb-option">
    <div>{{ item.label }}</div>
  </ng-container>
</nb-select>
```

This is currently possible to do and it works. Works here means it actually renders the `div`s correctly (they ofc do not work as an `option`, but that is expected).

I'm not sure if we should force the users of `nebular` to restrict them from using `ng-container` in this situation as probably some devs will prefer to use `ng-container` for loops and keep their `nb-options` cleaner.

Indeed doing this we're opening up the possibility for miss usage, but the same can happen for the pure html tags (`select` and `option`).

I'm basically opened to doing this any way you think it is best.